packages/protoc-gen-connect-query/src/generateTs.ts

```diff
@@ -95,11 +95,11 @@ const generateServiceFile =
  */
 export const generateTs: PluginInit["generateJs"] & PluginInit["generateTs"] = (
   schema,
-  extension,
+  extension
```

Comment:
Pretty sure prettier wants the comma.

Reply:
yeah weird, my local copy clearly didn't.